delocate/__init__.py

```diff
@@ -1,11 +1,12 @@
 # Init for delocate package
 
-from . import _version
 from .delocating import delocate_path, delocate_wheel, patch_wheel
 from .libsana import tree_libs, wheel_libs
 
-__version__ = _version.get_versions()["version"]
-del _version
+try:
```

Comment:
Sorry for my ignorance here, but under what circumstances would this be triggered?

Reply:
Not under normal circumstances, but if the module was imported directly from the repo without being installed in development mode then `_version.py` would be missing.  Versioneer seemed to have a fallback for the case when a version was asked for with the package being setup and this was me accounting for that with the new setup.

It should also be possible to cover this with a test which installs the dependencies but not the package itself. I'd also be okay with removing this try/except clause and just telling everyone to be sure to use `pip install -e .`

Reply:
How about raising a warning in the block, possibly a deprecation warning?

Reply:
A RuntimeWarning might be appropriate, but also simply crashing when `_version.py` is missing is also an option.  I'll go with whatever you want.

I think running the module without doing a development install is unrealistic and is going to crash on one of the imports of dependencies before it reaches this part.

I've had one of my own projects which required a trick like this but it was exceptional: It needed to import the package for doc generation but the package was too heavy to install on the doc generating servers.

Reply:
My preference would be to allow it for now, with a deprecation warning that we will break in the future - just because somebody may be using that mis-feature at the moment.